### PR TITLE
Modified HeapPage.java and HeapFile.java

### PR DESCRIPTION
**Pull request by Rithesh17**
I have implementer the InsertTuple and the DeleteTuple methods. The code didn't take much of my efforts and I finished them in an hour or so. I felt a small trouble in the deleteTuple method of HeapPage.java, but I finished it too. I have also implemented writePage method in HeapPage.

### CODE REVIEW

**Review by manishsbhoopalam — APPROVED** [review]

**Review by Ram-Aditya — APPROVED** [review]

**Review by Priiyam — APPROVED** [review]

**Comment by sbshah97** [comment]
Why has this Pull Request not been merged?

**Comment by Ram-Aditya** [comment]
@salman-bhai We thought we needed to wait for your approval as we had requested a review from your side.
Sorry about that!We `ll go ahead with the merge now then.

**Comment by Ram-Aditya** [comment]
Closing this pull request as updated one has been put up already.

**Pull request closed without merging**
